packages/cloud-cognitive/src/components/NonLinearReading/NonLinearReading.stories.js

```diff
@@ -0,0 +1,120 @@
+/**
+ * Copyright IBM Corp. 2023, 2023
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import { Link } from 'carbon-components-react';
+
+import {
+  getStoryTitle,
+  prepareStory,
+} from '../../global/js/utils/story-helper';
+
+import { NonLinearReading } from '.';
+import mdx from './NonLinearReading.mdx';
+
+import styles from './_storybook-styles.scss';
+
+export default {
+  title: getStoryTitle(NonLinearReading.displayName),
+  component: NonLinearReading,
+  parameters: {
+    styles,
+    docs: {
+      page: mdx,
+    },
+  },
+};
+
+const defaultProps = {
+  theme: 'light',
+};
+
+const Template = (args) => {
+  return <NonLinearReading {...args} />;
+};
+
+export const nonLinearReading = prepareStory(Template, {
+  args: {
+    children: 'Click to expand this "term"...',
```

Comment:
I think curly quotes are more appropriate, but pretty small nit. 🙈 

```suggestion
    children: 'Click to expand this “term”...',
```

Reply:
Had a conversation with Cameron Calder a while back: these are required. Thanks for pointing it out. I'll make the change.